doc/ref_cert/README.md

```diff
@@ -26,11 +25,11 @@ and all artifacts (reports, logs, etc.) to
 [an S3 compatible storage service](http://artifacts.opnfv.org/).
 
 The CNTT verification, validation, and conformance processes leverage
-existing OPNFV testing knowledge (projects) and experience (history) and then
-conform to the overall toolchain design already in-place. The RC toolchain
+existing OPNFV testing knowledge (projects) and experience (history) by leveraging
```

Comment:
The sentence use "leverage" twice As:
"The CNTT verification, validation, and conformance processes leverage existing OPNFV testing knowledge (projects) and experience (history) by leveraging the overall toolchain design already in-place."

Suggest replacing "leveraging" 
"The CNTT verification, validation, and conformance processes leverage existing OPNFV testing knowledge (projects) and experience (history) by utilising the OPNFV toolchain design already in-place."

```suggestion
existing OPNFV testing knowledge (projects) and experience (history) by utilising
```